jelly/src/main/java/org/kohsuke/stapler/jelly/IncludeTag.java

```diff
@@ -113,6 +113,10 @@ public void doTag(XMLOutput output) throws JellyTagException {
         } catch (RuntimeException e) {
             throw e;
         } catch (Exception e) {
+            if(optional) {
```

Comment:
What actual exception is thrown? How do you know it is safe to swallow this one?